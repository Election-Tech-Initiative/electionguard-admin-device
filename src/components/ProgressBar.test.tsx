import React from 'react'
import { render } from '@testing-library/react'

import ProgressBar from './ProgressBar'

it('renders ProgressBar with defaults', async () => {
  const { container } = render(<ProgressBar />)
  expect(container.firstChild).toMatchSnapshot()
})
