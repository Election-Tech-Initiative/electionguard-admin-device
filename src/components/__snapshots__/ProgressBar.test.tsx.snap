// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ProgressBar with defaults 1`] = `
<span
  class="sc-bdVaJa coUruJ"
/>
`;
